resolwe/flow/views/mixins.py

```diff
@@ -166,13 +166,6 @@ def bulk_delete(self, request, *args, **kwargs):
                 f"Model {to_delete.model} does not support background deletion."
             )
 
-        permissions_to_delete = set(to_delete.values_list("pk", flat=True))
```

Comment:
It is ok to skip objects that don't exist (user has no `VIEW` permission), but it should raise an error for objects with `VIEW` permission and no `EDIT` permission.